frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -359,6 +360,7 @@ local footerTextGeneratorMap = {
             if add_ellipsis then
                 fitted_chapter_text = fitted_chapter_text .. "…"
             end
+            fitted_chapter_text = fitted_chapter_text:gsub(" ", "\xC2\xA0") -- replace space with no-break-space
```

Comment:
Well, at least do it similarly as for title :/
(There, you're doing it too late, after the text has been measured and truncated - so, in case in some font a space and a no-break-space have different widths, there might be strange things.)

Reply:
@poire-z Thank you. Does this commit look better?